docs/hello_nextflow/seqera/01_run_with_cli.md

```diff
@@ -0,0 +1,167 @@
+## 1. Use Seqera Platform to capture and monitor Nextflow jobs launched from the CLI
+
+We'll start by using the Nextflow CLI to launch a pipeline and monitor it in Seqera Platform.
+Start by logging into the [Seqera Platform](https://cloud.seqera.io/).
+
+!!! info "Nextflow Tower"
+
+    Seqera Platform was previously known as Nextflow Tower.
+    You'll still see references to the previous name in environment variable and cli option names.
+
+### 1.1. Set up your Seqera Platform token by exporting it to your environment
+
+Follow these steps to set up your token:
+
+1.  Create a new token by clicking on the **Settings** drop-down menu:
+
+    ![Create a token](seqera/img/usage_create_token.png)
```

Comment:
The images are a bit small, see if you can increase the zoom level to make them more readable. (ok to leave as a future improvement)

Reply:
Will leave for future improvement.